src/uct/cuda/cuda_ipc/cuda_ipc_md.c

```diff
@@ -25,6 +26,16 @@ static ucs_config_field_t uct_cuda_ipc_md_config_table[] = {
     {"", "", NULL,
      ucs_offsetof(uct_cuda_ipc_md_config_t, super), UCS_CONFIG_TYPE_TABLE(uct_md_config_table)},
 
+    {"RCACHE", "try", "Enable using memory registration cache",
+     ucs_offsetof(uct_cuda_ipc_md_config_t, rcache_enable), UCS_CONFIG_TYPE_TERNARY},
+
+    {"", "", NULL,
```

Comment:
Having infinite number of regions by default is ok because most use cases don't run into memory leak issue. For the cases that have this issue, they have to limit either number of regions or total size of rcache regions (either by using max_ratio or max_size)